paymentsheet/src/test/java/com/stripe/android/customersheet/CustomerSessionViewModelTest.kt

```diff
@@ -39,6 +39,25 @@ class CustomerSessionViewModelTest {
         assertThat(component1).isNotEqualTo(component2)
     }
 
+    @Test
+    fun `createCustomerSessionComponent creates a new session if single component is different`() {
```

Comment:
Is this supposed to be this?

```suggestion
    fun `createCustomerSessionComponent creates a new session if configuration changes`() {
```